snowfakery/standard_plugins/datasets.py

```diff
@@ -119,10 +104,12 @@ def query(self):
 
 
 class CSVDatasetLinearIterator(DatasetIteratorBase):
-    def __init__(self, datasource: Path):
+    def __init__(self, datasource: Path, repeat: bool):
         self.datasource = datasource
         self.file = open(self.datasource, newline="", encoding="utf-8-sig")
         self.start()
+        self.result = vars(self)  # FIXME
```

Comment:
What fix is needed here? I've never used `vars` before and don't quite understand what is going on here.

Reply:
Deleted that line. I used it for debugging at some point. Thanks for noticing.

https://github.com/SFDO-Tooling/Snowfakery/pull/457/commits/b7054d2b2adf278da6ba5fbcc1be10ebcb9aa76a#diff-bbb25a17a93d6547a2a47f8e856fafcf4acc4571a1b9d6bdc8fb6e609f7e6077L111